functions/dcWaiter.R

```diff
@@ -31,28 +33,49 @@ dcWaiter <- function(stage = c("show", "update", "hide"),
         ),
         color = "#424874"
       )
-    } else if (isPass) {
+    # } else if (!isLogin) {
+    #   # when user is not login
+    #   waiter_update(html = tagList(
+    #     img(src = "img/synapse_logo.png", height = "120px"),
+    #     h3("Looks like you're not logged in!"), 
+    #     span("Please ", 
+    #       a("login", href = "https://www.synapse.org/#!LoginPlace:0", target = "_blank"),
+    #       " to Synapse, then refresh this page."
+    #     )
+    #   ))
+    } else if (!isCertified) {
+      # when user is not certified synapse user
       waiter_update(html = tagList(
         img(src = "img/synapse_logo.png", height = "120px"),
-        h3(sprintf("Welcome, %s!", usrName))
+        h3("Looks like you're not a certified synapse user!"),
+        span("Please follow the ", 
+          a("instruction", 
+            href = "https://help.synapse.org/docs/User-Account-Tiers.2007072795.html#UserAccountTiers-CertifiedUsers", 
+            target = "_blank"
+          ),
+          " to become a certified user, then refresh this page."
+        )
+      ))
```

Comment:
@milen-sage 

Since I was improving the initial loading (landing) screen, the approach in this pr is to stop users to see the app if they are not certified synapse users. It will stop at a loading screen and redirect users to certify.

But if you prefer that uncertified users still be able to use the app, I can just add validation before the submission.